src/Compilation/Config.zig

```diff
@@ -302,6 +302,12 @@ pub fn resolve(options: Options) ResolveError!Config {
             break :b false;
         }
 
+        if (target.cpu.arch.isLoongArch()) {
+            // FIXME:
+            // loongarch LTO might have problem on LLVM 18.
```

Comment:
This needs more details to be mergeable: What exactly is the problem? Is it expected to be fixed in 19?

Reply:
I checked the source again. The problem was in GCC not LLVM.

https://gcc.gnu.org/bugzilla/show_bug.cgi?id=113233

Removed this code.